packages/color-slider/src/ColorSlider.ts

```diff
@@ -286,7 +288,7 @@ export class ColorSlider extends Focusable {
     }
 
     private get handlePositionStyles(): string {
-        return `${this.vertical ? 'top' : 'left'}: ${
+        return `${this.vertical ? 'top: unset; bottom' : 'left'}: ${
```

Comment:
You've got `top: unset` in the CSS, can we remove it here?

Reply:
Yes, we can. This was added out of undo caution.